src/Features/OnboardingTasks/Tasks/WooCommercePayments.php

```diff
@@ -143,22 +144,12 @@ public static function is_connected() {
 	 * @return bool
 	 */
 	public static function is_supported() {
-		return in_array(
-			WC()->countries->get_base_country(),
-			array(
-				'US',
-				'PR',
-				'AU',
-				'CA',
-				'DE',
-				'ES',
-				'FR',
-				'GB',
-				'IE',
-				'IT',
-				'NZ',
-			),
-			true
-		);
+		$suggestions = Suggestions::get_suggestions();
+		$woocommerce_payments_ids = preg_grep( '/^woocommerce_payments/', array_column( $suggestions, 'id' ) );
```

Comment:
Should we use the `plugins` property here to check if `woocommerce-payments` is installable?  Seems like it might be a bit more future-proof over IDs.

Reply:
Good suggestion, I debated on it, but then decided not to, given it wasn't quite as easy. But it does make more sense.
I will make this change in the `woocommerce` repo, given @octaedro is merging in the simpler fix for `3.3`.